hoa2dot/parsers.py

```diff
@@ -160,7 +163,7 @@ def headername(self, args):
 
     def body(self, args):
         """Parse the 'body' node."""
-        state2edges = OrderedDict({})  # type: Dict[State, List[Edge]]
+        state2edges: Dict[State, List[Edge]] = OrderedDict({})
```

Comment:
this is not backward compatible with Python 3.6 and lower. I suggest using the old annotation.

Reply:
ook, I didn't know